lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_vmdb.rb

```diff
@@ -7,8 +7,6 @@ def vmdb(model_name, *args)
 
     def service_model(model_name)
       "MiqAeMethodService::MiqAeService#{model_name}".constantize
-    rescue NameError
-      service_model_lookup(model_name)
```

Comment:
I don't like that the original code used NameError for the lookup...since you are fixing it, can you make it just go through the lookup code always and if there is no value to be looked up, it defaults to "MiqAeMethodService::MiqAeService#{model_name}".constantize